app/generate-index/index.ts

```diff
@@ -0,0 +1,32 @@
+import { bulkUploadData } from "./add-data-to-atlas";
+import { genIndex } from "./gen-index";
+import * as dotenv from "dotenv";
+dotenv.config({ path: ".env.local" });
+
+const { DB_NAME, COLLECTION_NAME } = process.env;
+
+const sitesToIndex = [
```

Comment:
note: right now i just have it indexing realm and app services. we can add N other sites later. i just wanted to see how it'd work on these.